package.json

```diff
@@ -85,6 +81,10 @@
     "@types/node": "^10.12.17",
     "@typescript-eslint/eslint-plugin": "^1.3.0",
     "@typescript-eslint/parser": "^1.3.0",
+    "@chainsafe/ssz": "^0.1.2",
+    "@chainsafesystems/ssz": "^1.6.0",
+    "@nodeutils/defaults-deep": "^1.1.0",
+    "@polkadot/ts": "^0.1.56",
```

Comment:
oh there was misunderstanding, I meant to move `@polakdot/ts` only because those are typings that aren't used once the library is built.